src/Ombi.Notifications/Agents/DiscordNotification.cs

```diff
@@ -126,6 +126,8 @@ protected override bool ValidateConfiguration(DiscordNotificationSettings settin
                 Message = parsed.Message,
             };
             notification.Other.Add("image", parsed.Image);
+            notification.Other.Add("author", "New Issue!");
```

Comment:
This should be the subject, located on `notification.Subject`

Reply:
My bad, I changed how that worked, and then changed it back to keep the original functionality (but clearly did it wrong). On it.